src/Lykke.Service.BitfinexAdapter.Services/OrderBooksHarvester/OrderBooksHarvesterBase.cs

```diff
@@ -97,12 +95,12 @@ protected void RechargeHeartbeat()
             while (!CancellationToken.IsCancellationRequested)
             {
                 var msgInSec = _lastSecPublicationsNum / period;
-                var pubInSec = _publishedToRabbit / period;
+                var pubInSec = _totalOrderbooksPublishedToRabbit / period;
                 await Log.WriteInfoAsync(nameof(OrderBooksHarvesterBase),
-                    $"Receive rate from {ExchangeName} {msgInSec} per second, publish rate to " +
-                    $"RabbitMq {pubInSec} per second", string.Empty);
+                    $"Receive rate from {Constants.BitfinexExchangeName} {msgInSec} per second, publish rate to " +
+                    $"RabbitMq {pubInSec} per second.", string.Empty);
```

Comment:
Is it correct overload of WriteInfo? Seems like you log empty string.

Reply:
Done. Log message used placed at the wrong place - its a valid overload but context and info params were swapped.